### PR TITLE
paper: Add Quantum Machine Learning in High Energy Physics

### PR DESCRIPTION
**Pull request by vlimant**
Add reference for [Quantum Machine Learning in High Energy Physics](https://inspirehep.net/literature/1796743) and add DOI for Graph neural networks in particle physics.

```
* Add reference for Quantum Machine Learning in High Energy Physics
   - c.f. https://inspirehep.net/literature/1796743
* Add DOI for "Graph neural networks in particle physics" for journal publication
```

make_md.py failed in the master with 

```pytb
Traceback (most recent call last):
  File "make_md.py", line 121, in <module>
    myfile_out.write(mybuffer+"    * "+convert_from_bib(cite)+"\n")
  File "make_md.py", line 21, in convert_from_bib
    for line in myfile_bib:
  File "/usr/lib64/python3.6/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 7111: ordinal not in range(128)
```

hence the absence of update on README.md in this PR

### CODE REVIEW

**Comment by matthewfeickert** [comment]
Thanks for your PR @vlimant. :+1:  It is late for me in the US (and I'm just seeing this now I'm afraid) so I will take a look at this once I'm out of meetings in the morning.

File: HEPML.bib [hunk 1]
```diff
@@ -1170,13 +1170,19 @@ @article{1808859
...
 }
 @article{1808887,
-    author = "Shlomi, Jonathan and Battaglia, Peter and Vlimant, Jean-Roch",
-    title = "{Graph Neural Networks in Particle Physics}",
-    eprint = "2007.13681",
-    archivePrefix = "arXiv",
-    primaryClass = "hep-ex",
-    month = "7",
-    year = "2020"
+   title={Graph neural networks in particle physics},
```
Comment (matthewfeickert) [inline HEPML.bib:1173]:
```suggestion
   title="{Graph neural networks in particle physics}",
```

Comment (matthewfeickert) [reply HEPML.bib:1173]: Oh @vlimant, this is an easy fix I think. Can you just quote the title?

**Review by matthewfeickert — CHANGES_REQUESTED** [review]
@vlimant I'll still review this more in my late-morning, but please check out this suggested fix.

**Comment by vlimant** [comment]
quoted the title. the issue seems to pop up at

> @article{Grossi:2020orx,
>     author = "Grossi, M. and Novak, J. and Rebuzzi, D. and Kersevan, B.",
>     title = "{Comparing Traditional and Deep-Learning Techniques of Kinematic Reconstruction for polarisation Discrimination in Vector Boson Scattering}",
>     eprint = "2008.05316",
>     archivePrefix = "arXiv",
>     primaryClass = "hep-ph",
>     month = "8",
>     year = "2020"
> }

in the HEPML.bib though

**Comment by matthewfeickert** [comment]
@vlimant I'm unable to reproduce your error:

```
$ docker run --rm -ti python:3.6 /bin/bash
root@eb5eaa629798:/# git clone https://github.com/vlimant/HEPML-LivingReview.git
Cloning into 'HEPML-LivingReview'...
remote: Enumerating objects: 38, done.
remote: Counting objects: 100% (38/38), done.
remote: Compressing objects: 100% (26/26), done.
remote: Total 537 (delta 20), reused 24 (delta 11), pack-reused 499
Receiving objects: 100% (537/537), 430.14 KiB | 2.85 MiB/s, done.
Resolving deltas: 100% (366/366), done.
root@eb5eaa629798:/# cd HEPML-LivingReview/
root@eb5eaa629798:/HEPML-LivingReview# git checkout origin/doi_updates -b doi_updates
Branch 'doi_updates' set up to track remote branch 'doi_updates' from 'origin'.
Switched to a new branch 'doi_updates'
root@eb5eaa629798:/HEPML-LivingReview# python make_md.py 
root@eb5eaa629798:/HEPML-LivingReview# git diff --name-only
README.md
root@eb5eaa629798:/HEPML-LivingReview#
```

indicating there is either something wrong with your local setup or your Python runtime (aside you might want to move off Python 3.6 as [its end of life is 2021-12](https://www.python.org/dev/peps/pep-0494/#lifespan)). Can you run `git clean` and try again?

**Review by matthewfeickert — APPROVED** [review]
@vlimant I'm still not sure what the issue was with your CPython 3.6 install, but things work fine for me and CI once I run `make_md.py`, so after rebasing this it LGTM. :+1: Thanks very much for your contribution &mdash; we truly appreciate it &mdash; and congratulations on the paper as well!

**Pull request merged**